admin/pages/help-settings.php

```diff
@@ -17,14 +17,21 @@
 
 $wi_form = new WI_Volunteer_Management_Form();
 $wi_form->admin_header();
+
+// Filter to allow the Help & Settings menu 'Help' tab to be hidden
+$wivm_show_help_tab	= apply_filters( 'wivm_show_help_tab', true );
+
 ?>
-	
+
 	<h2 class="nav-tab-wrapper" id="wivm-tabs">
-		<a class="nav-tab" id="help-tab" href="#top#help"><span class="dashicons dashicons-editor-help"></span> <?php _e( 'Help', 'wired-impact-volunteer-management' ); ?></a>
+		<?php if( $wivm_show_help_tab === true ) : ?>
+			<a class="nav-tab" id="help-tab" href="#top#help"><span class="dashicons dashicons-editor-help"></span> <?php _e( apply_filters( 'wivm_help_tab_name', 'Help' ), 'wired-impact-volunteer-management' ); ?></a>
```

Comment:
Can you give me an idea of the use case for using the `wivm_help_tab_name` filter? I'm not sure how useful that would be, but I don't mind including it if you think others might use it. If we do include it though it probably makes sense to filter the other tab names as well. If we do that, it might make sense to store the tab names and details in an array, filter them, then use a loop to output the HTML. That way we keep it dry and easy to filter.

Reply:
Hey JG, as I mentioned on our call I'm going to get this filter removed.